Cargo.toml

```diff
@@ -31,6 +31,9 @@ unindent = { version = "0.2.1", optional = true }
 # support crate for multiple-pymethods feature
 inventory = { version = "0.3.0", optional = true }
 
+# coroutine implementation
+futures = { version = "0.3", optional = true }
```

Comment:
Do we need all of `futures` or just `futures-task`?

Reply:
Right, only `futures-task` is needed.